source/universal/components/Dashboard/index.js

```diff
@@ -9,15 +9,16 @@ class Dashboard extends Component {
   static contextTypes = {
     logout: PropTypes.func.isRequired,
     syncRepos: PropTypes.func.isRequired,
+    addTag: PropTypes.func.isRequired,
```

Comment:
FYI, I just updated the way we require actions creators. Now you have to use [`context` API of react](http://facebook.github.io/react/docs/context.html) to pass down action creators. Implementation in this repo can be found [here](https://github.com/d6u/github-starboard/blob/master/source%2Fclient%2FApp.js#L9-L19)